src/test/test-modifier.ts

```diff
@@ -45,15 +45,12 @@ const resources = `${__dirname}/../../src/test/resources`;
             // go!
             const codeModel = await modeler.process();
 
-            // console.log(serialize(codeModel))
-            const yaml = serialize(codeModel);
             const fileName = `${__dirname}/../../src/test/resources/` + each + "/" + each + `-az-modifier.yaml`;
-            //await (writeFile(fileName, yaml));
```

Comment:
this two lines better reserve. they will be helpful when there's change in namer or modifier.